components/selection/selection-header.js

```diff
@@ -172,6 +163,18 @@ export class SelectionHeader extends SelectionObserverMixin(RtlMixin(LocalizeCor
 		this._hasActions = (e.target.assignedNodes({ flatten: true }).filter(node => node.nodeType === Node.ELEMENT_NODE).length > 0);
 	}
 
+	_renderSelection() {
```

Comment:
My plan was to keep this as the default implementation. The `d2l-selection-header` currently works as-is, making it empty/abstract would break that.

I know there was some debate about whether it's desirable or not for `d2l-selection-header` to exist standalone, though I can say that it's used directly in our selection demos (on the readme and the internal dev demos). It was a perfect drop-in replacement for the custom-made "selection header" we had in the examples previously.